examples/iframe.html

```diff
@@ -46,6 +46,7 @@
   iframe.src = 'index.html';
   iframe.width = '100%';
   iframe.height = '100%';
+  iframe.setAttribute('allow', 'gyroscope,accelerometer');
```

Comment:
Is this the correct feature policy?

Reply:
shouldn't it be "accelerometer; gyroscope" ?

Reply:
It's surprisingly difficult to find documentation on this :) having trouble remote debugging on Chrome m65 (where permissions should be enabled) so will use that to test

Reply:
After some testing, both `,` and `;` seemed to work